q2_feature_classifier/classifier.py

```diff
@@ -133,20 +157,27 @@ def classify(reads: DNAIterator, classifier: Pipeline,
 
 
 _classify_parameters = {'chunk_size': Int, 'n_jobs': Int, 'pre_dispatch': Str,
-                        'confidence': Float}
+                        'confidence': Float, 'force_orientation':
+                        Str % Choices(['same', 'reverse-complement'])}
 
 
 plugin.methods.register_function(
-    function=classify,
+    function=classify_sklearn,
     inputs={'reads': FeatureData[Sequence],
             'classifier': TaxonomicClassifier},
     parameters=_classify_parameters,
     outputs=[('classification', FeatureData[Taxonomy])],
-    name='Classify reads by taxon.',
+    name='Pre-fitted sklearn-based taxonomy classifier',
     description='Classify reads by taxon using a fitted classifier.',
     parameter_descriptions={'confidence': 'Confidence threshold for limiting '
-                            'taxonomic depth. Negative value disables. '
-                            'Currently experimental. USE WITH CAUTION.'}
+                            'taxonomic depth. Negative value disables '
+                            'Currently experimental. USE WITH CAUTION',
+                            'force_orientation': 'Force assumption that reads '
```

Comment:
I'm not in love with the name `force_orientation`; the word "force" seems unnecessarily strong here, especially when `"same"` is passed -- there's no "forcing" in that case, the reads are classified as usual without transformation. The default case (autodetection) also makes "forcing" seem out of context here.

Since the user is describing the orientation their input reads w.r.t the reference, what about `read_orientation` (I suggested this [here](https://github.com/qiime2/q2-feature-classifier/pull/62#discussion_r110752081))?